### ISSUE TITLE
udisks - usb external hdd

### ISSUE THREAD

**Issue by seancmalloy** [jcorporation/myMPD#1035]
its been a minute since i used mympd but it can no longer see my mounted usb external hdd using neighbors - udisks. any ideas i cant find much information on it online.

**Comment by jcorporation** [comment]
Can you see it with mpc?
Please add debug logs of myMPD and MPD.

**Comment by seancmalloy** [comment]
mpc? 

i found error in logs stating "enabled" is unknown in line for neighbors - udisks

neighbors  {
     plugin           "udisks"
     enabled          "yes"
}

is  enabled  "yes" still required? when i remove it no error, but still cant see content

**Comment by jcorporation** [comment]
Read mpc documentation how to list neighbors with it. If this works with mpc, it‘s a myMPD bug, else a mpd bug or misconfiguration.

**Comment by seancmalloy** [comment]
you are correct something is broken. started fresh noticed that latest mympd requires newer mpd than debain 11 provides had to compile. instead of compiling mympd tried your debian repo installed with apt. but now it seems it cant find mympd.conf, this config file is still used correct?

**Comment by jcorporation** [comment]
No, mympd.conf no longer exists. There are now single files in /var/lib/mympd/config.

myMPD support MPD 0.21 and above: https://jcorporation.github.io/myMPD/references/feature-matrix
